packages/hot-module-replacement/client.js

```diff
@@ -1,28 +1,28 @@
 // TODO: add an api to Reify to update cached exports for a module
-const ReifyEntry = require('/node_modules/meteor/modules/node_modules/@meteorjs/reify/lib/runtime/entry.js')
+var ReifyEntry = require('/node_modules/meteor/modules/node_modules/@meteorjs/reify/lib/runtime/entry.js')
```

Comment:
Why everything is this file is now `var` instead of `const` and `let`?

Reply:
if you use const or any ES6 it breaks the IE11, we have a check for it now

Reply:
Oh, I see... lol